frontend/src/components/landing-page/landing-page.tsx

```diff
@@ -37,19 +39,24 @@ export const LandingPage = (): React.ReactElement => {
               <br />
               Both new & used. Find the car of your dreams with us!
             </div>
-            <div className={styles.firstContainerLeftButtons}>
-              <Link to={AppRoute.SIGN_UP}>
-                <ButtonFill
-                  className={clsx(styles.button, styles.buttonFill)}
-                  text="Create Account"
-                />
-              </Link>
-              <Link to={AppRoute.SIGN_IN}>
-                <ButtonOutline
-                  className={clsx(styles.button, styles.buttonOutline)}
-                  text="Sign In"
-                />
-              </Link>
+            {!isAuth && (
+              <div className={styles.firstContainerLeftButtons}>
+                <Link to={AppRoute.SIGN_UP}>
+                  <ButtonFill
+                    className={clsx(styles.button, styles.buttonFill)}
+                    text="Create Account"
+                  />
+                </Link>
+                <Link to={AppRoute.SIGN_IN}>
+                  <ButtonOutline
+                    className={clsx(styles.button, styles.buttonOutline)}
+                    text="Sign In"
+                  />
+                </Link>
+              </div>
+            )}
+            <div className={styles.searchWrapper}>
+              {isAuth && <SimpleAutoFilter />}
             </div>
```

Comment:
Don't we need:
```suggestion
           {isAuth && <div className={styles.searchWrapper}>
               <SimpleAutoFilter />}
            </div>}
```

Because why should an empty `div` hang in the DOM?